container-images/scripts/llama-server.sh

```diff
@@ -21,6 +21,7 @@ eval llama-server \
      --host "${HOST:=0.0.0.0}" \
      --port "${PORT:=8001}" \
      --gpu_layers "${GPU_LAYERS:=0}" \
+     --ctx_size "${CTX_SIZE:=2048}" \
```

Comment:
We should not assume a default size as llama-cpp uses 4096 by default.

Reply:
This is really the default ramalama is using:

https://github.com/containers/ramalama/blob/ab7e7594fbbb7819b5da10a1dd8ab2cbfe349f32/ramalama/config.py#L79

I'm setting a default value to make the env variable optional.

The other options is to make the script a little bit more involved and to allow it to be optional. I'll add that in a subsequent commit.

Reply:
Ollama uses 2048, everyone chooses different defaults, it's ok